public/gpudevice.h

```diff
@@ -129,6 +129,38 @@ class GpuDevice : public HWCThread {
                     std::vector<std::vector<uint32_t>>& panorama_sos_displays,
                     std::vector<bool>& available_displays);
 #endif
+  void ParseLogicalDisplaySetting(std::string& value,
+                                  std::vector<uint32_t>& logical_displays);
+  void ParseMosaicDisplaySetting(
+      std::string& value, std::vector<std::vector<uint32_t>>& mosaic_displays);
+  void ParsePhysicalDisplaySetting(std::string& value,
+                                   std::vector<uint32_t>& physical_displays);
+  void ParseCloneDisplaySetting(
+      std::string& value, std::vector<std::vector<uint32_t>>& cloned_displays);
+  void ParsePhysicalDisplayRotation(
+      std::string& value, std::vector<uint32_t>& display_rotation,
+      std::vector<uint32_t>& rotation_display_index);
+  void ParseFloatDisplaySetting(std::string& value,
+                                std::vector<HwcRect<int32_t>>& float_displays,
+                                std::vector<uint32_t>& float_display_indices);
+
+  void DisplayInit(std::vector<uint32_t>& physical_displays,
```

Comment:
the Naming should be InitializeXxxx, same as ParseXxxx. as above.

Reply:
Thanks yuanjun, i refer to the Wan Shuang's:
#ifdef ENABLE_PANORAMA
  if (use_panorama && !use_mosaic && !use_cloned && !use_float) {
    **PanoramaInit**(total_displays_, temp_displays, panorama_displays,
                 panorama_sos_displays, available_displays);
  }
#endif

Does all the init function need change to InitializeXXX? Including the PanoramaInit？

Reply:
yes we shall better align with the same naming style.

Reply:
ok, thanks!